airflow/__init__.py

```diff
@@ -79,6 +80,7 @@ def __init__(self, namespace):
 from airflow import macros
 from airflow import contrib
 
-operators.integrate_plugins()
-hooks.integrate_plugins()
-macros.integrate_plugins()
+operators._integrate_plugins()
+hooks._integrate_plugins()
+executors._integrate_plugins()
```

Comment:
@jasminetsai @jiffyclub we can make our own scheduler
https://airflow.incubator.apache.org/code.html?highlight=executors#executors